Create ui-c8y-1021-57-0-add-c-8-y-is-device-marker-to-bus-devices-8614.md (#3158)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y-1021-57-0-add-c-8-y-is-device-marker-to-bus-devices-8614.md

---
date: ""

> title: add c8y_IsDevice marker to bus-devices (#8614) [GRAFT][release/cd] (#8784)

Reviewer: ```suggestion
title: Modbus, Profibus, CANBus, and CANopen devices are now marked with c8y_IsDevice fragment
```

> title: Modbus, Profibus, CANBus, and CANopen devices are now marked with c8y_IsDevice fragment

Reviewer 2: ```suggestion
title: c8y_IsDevice fragment now added to Modbus, Profibus, CANBus, and CANopen devices
```

product_area: Device management & connectivity
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-4557
version: 1021.57.0
---

> add c8y_IsDevice marker to bus-devices (#8614) [GRAFT][release/cd] (#8784)

Reviewer: In the past, Modbus, Profibus, CANBus, and CANopen devices created by the device management application were missing the c8y_IsDevice marker which identifies them as devices in the inventory. This change adds the c8y_IsDevice fragment to all bus devices created going forward. Existing bus devices in the inventory are not modified and will still be missing the marker. This improvement makes these devices easier to find and filter in the inventory when querying for devices.

> add c8y_IsDevice marker to bus-devices (#8614) [GRAFT][release/cd] (#8784)

Reviewer: ```suggestion
In the past, Modbus, Profibus, CANBus, and CANopen devices created by the device management application were missing the c8y_IsDevice marker which identifies them as devices in the inventory. This change adds the c8y_IsDevice fragment to all bus devices created going forward. Existing bus devices in the inventory are not modified and will still be missing the marker. This improvement makes these devices easier to find and filter in the inventory when querying for devices.
```

> In the past, Modbus, Profibus, CANBus, and CANopen devices created by the device management application were missing the c8y_IsDevice marker which identifies them as devices in the inventory. This change adds the c8y_IsDevice fragment to all bus devices created going forward. Existing bus devices in the inventory are not modified and will still be missing the marker. This improvement makes these devices easier to find and filter in the inventory when querying for devices.

Reviewer 2: ```suggestion
In the past, Modbus, Profibus, CANBus, and CANopen devices created in the Device Management application were missing the `c8y_IsDevice` fragment which identifies them as devices in the inventory. This change adds the `c8y_IsDevice` fragment to all such devices created going forward. This improvement makes such devices easier to find and filter in the inventory when querying for devices. Existing Modbus, Profibus, CANBus, and CANopen devices in the inventory are not modified and will still be missing the parameter. 
```
